wait-for-it.sh

```diff
@@ -13,6 +13,9 @@ Usage:
     -h HOST | --host=HOST       Host or IP under test
     -p PORT | --port=PORT       TCP port under test
                                 Alternatively, you specify the host and port as host:port
+    -c COMMAND | --command=COMMAND 
+                                A single command to test
+    -w TIMEOUT | --wait=TIMEOUT Wait / sleep time between each test attempt
```

Comment:
`wait` is a very confusing choice of wording in this context. Maybe call it something like `interval` instead?

Reply:
I agree, I renamed the name of the option.